### DISCUSSION TITLE
Release v5.5.3 for GPUArray v11?

### DISCUSSION THREAD

**Discussion by milankl** [JuliaGPU/CUDA.jl#2588] [General]
Tim, would it be possible to release v5.5.3 to have a tagged version that has compatibility with GPUArrays v11?

**Comment by maleadt** [comment]
Not yet, no. There are still several KA.jl-related regressions we need to look into.

Why do you require GPUArrays v11?

**Reply by maximilian-gelbrecht** [reply]
With the syntax changes in GPUArrays, we already adjusted our usage of it a few weeks after v11 was released. 

To be honest, I just didn't think about it being incompatible with CUDA.jl. 

Can you give a rough estimate when a compatible version would be released? Just to know if it's worth it to roll back the changes we made in our package.

**Comment by maleadt** [comment]
The problem is really finding time to look into the regressions. I currently have some other obligations, and @vchuravy doesn't appear to have time either. I hope this to be resolved in a couple of weeks (early in the new year, realistically), but if you need this merged feel free to help out with the optimization work: https://github.com/JuliaGPU/GPUArrays.jl/issues/565 (there's several PRs on KA.jl too)
